src/diffusers/models/embeddings.py

```diff
@@ -714,6 +714,58 @@ def forward(self, ids: torch.Tensor) -> torch.Tensor:
         return freqs_cos, freqs_sin
 
 
+class CogView3PlusPatchEmbed(nn.Module):
```

Comment:
(nit): Since we now have a couple of patch embedding methods for the image generation models, it'd be nice to perhaps include a short docstring to brief about how this is different from them here?

Reply:
I think many of our embeds miss documentation. Better to tackle in separate PR (perhaps good community issue as well for hacktoberfest?)